static/js/show-task.js

```diff
@@ -37,3 +37,21 @@ function saveTask() {
             );
     }});
 }
+
+$(window).on('load', function worker() {
+            search_term = document.getElementById("search_by_task_name_form").search_term.value
+            worklist_name = document.getElementById("search_by_task_name_form").worklist_name.value
+            worklist_created_by = document.getElementById("search_by_task_name_form").worklist_created_by.value
+
+            $('#task-table').html('');
+            $('#task-table').addClass('loader');
+            $('#task-table').html('').load(
+            "/worklist-tool/update-task-table?search_term=" + search_term + "&worklist_name=" +
```

Comment:
I see we're trying to preserve the search term at each refresh, which is fine I think, but it looks like it interacts poorly with making a search by hand.

Steps to reproduce:

1- Open a worklist, type "aaaaa" in the search box.
2- Click the search button.

Expected result: I get a message telling me there are no results.
Actual results: The page reloads, the "no result" message flickers on screen but my search term is lost, so this worker soon makes and empty search and re-populates the table with all tasks.